packages/eslint-plugin-pf-codemods/lib/rules/v5/onToggle-updated-paramaters.js

```diff
@@ -0,0 +1,87 @@
+const { getPackageImports } = require("../../helpers");
+
+// https://github.com/patternfly/patternfly-react/pull/8667
+module.exports = {
+  meta: { fixable: "code" },
+  create: function (context) {
+    const onToggleAPIUpdateList = [
+      "ApplicationLauncher",
+      "BadgeToggle",
+      "DropdownToggle",
+      "KebabToggle",
+      "Toggle",
+      "Select",
+      "SelectToggle",
+    ];
+    const imports = getPackageImports(context, "@patternfly/react-core").filter(
+      (specifier) => onToggleAPIUpdateList.includes(specifier.imported.name)
+    );
+
+    return imports.length === 0
+      ? {}
+      : {
+          JSXOpeningElement(node) {
+            const onToggleProp = node.attributes.find(
+              (attr) => attr.name.name === "onToggle"
+            );
+
+            const { type, params, name } =
+              onToggleProp?.value?.expression || {};
+
+            if (
+              imports.map((imp) => imp.local.name).includes(node.name.name) &&
+              ((type === "ArrowFunctionExpression" && params?.length === 1) ||
+                type === "Identifier")
+            ) {
+              context.report({
+                node,
```

Comment:
think we could strip this out to be reused? seems like it will be in several places already

Reply:
i see you left a comment regarding this, i'll leave that up to you